File: samples/test/sumProd/SumProdC/S3e.c
 void sumProd(int n) {
	float sum = 0.0;//C1
	float prod = 1.0;
	int i;
	for (i = 1; i <= n; i++) {
		if (n % 2 == 0)	sum = sum + i; 
		prod = prod * i;
		foo(sum, prod);
	}
}

File: samples/test/sumProd/sumProdJS/S3e.js
function sumProd(n) {
	var sum = 0.0;//C1
	var prod = 1.0;	
	for (var i = 1; i <= n; i++) {
		if (i % 2 == 0) sum += i;
		prod = prod * i;
		foo(sum, prod);
	}
}